overseerr-api.yml

```diff
@@ -1,7 +1,7 @@
-openapi: '3.0.2'
+openapi: "3.0.2"
```

Comment:
We don't need changes to double quotes here. Can you revert these changes and only add your changes to the spec?

Reply:
Prettier doint its magic, commit coming